src/Components/MinimalCtaBanner.tsx

```diff
@@ -1,6 +1,8 @@
 import { Sans } from "@artsy/palette"
 import React from "react"
+import track from "react-tracking"
```

Comment:
We should use our typed version of `track` here
https://github.com/artsy/reaction/blob/master/src/Artsy/Analytics/track.ts

and add approriate values to the schema (https://github.com/artsy/reaction/blob/master/src/Artsy/Analytics/Schema/Values.ts#L70) if needed